### PR TITLE
Autoadmin Config

### PR DESCRIPTION
**Pull request by AnturK**
Simple autoadmin config for test servers and such.
Probably needs more !!TEST ONLY!! warnings

### CODE REVIEW

**Comment by Jalleo** [comment]
:moneybag: :moneybag: :money_with_wings:

File: code/modules/client/client procs.dm [hunk 1]
```diff
@@ -107,6 +107,17 @@ var/next_external_rsc = 0
 	directory[ckey] = src
 
 	//Admin Authorisation
+	if(config.autoadmin)
+		if(!admin_datums[ckey])
+			var/datum/admin_rank/autorank
+			for(var/datum/admin_rank/R in admin_ranks)
+				if(R.name == config.autoadmin_rank)
+					autorank = R
```
Comment (Giacom) [inline code/modules/client/client procs.dm]: You could put a break here to stop the loop from searching after it has found the rank.

Comment (AnturK) [inline code/modules/client/client procs.dm]: Huh, i'm pretty sure i added one there.. Must have overwritten it when adding the null check.. Will fix in a sec.

**Comment by optimumtact** [comment]
Thanks man :+1:

**Comment by Shadowlight213** [comment]
You should probably try and exclude the autoadmin config from being accessed by the debug configuration controller verb if its possible. Possibly make it require +permissions to modify.

**Comment by Jordie0608** [comment]
Since it's for easier local testing and whatnot the config should default to being on and give the Game Master rank rather than Game Admin.

**Pull request merged**